_data/developerpeachy/developerpeachy.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Rosalie Prom
+institution: University of Technology Sydney 🚩
+profile_pic: headshot.jpg # Name and extension of your profile picture(ex. mona.png)
+quote: "Can we add a design system here?" # no longer than 100 characters
```

Comment:
Please do not include quotes

Reply:
thanks for checking, it should be fixed now :)